install_sct.bat

```diff
@@ -0,0 +1,45 @@
+:: Usage: install_sct.bat <version>
+:: e.g.
+::        install_sct.bat 5.5
+
+:: Fetch version number (should be git branch, git tag, etc.)
+set git_ref=%1
+
+:: Save present working directory to return to later
+FOR /F "tokens=*" %%G IN ('cd') do (SET PWD=%%G)
+
+:: Go to user's home directory
+cd %HOMEPATH%
+
+:: Download SCT and check out the branch requested by the user
+git clone -b %git_ref% --single-branch --depth 1 https://github.com/spinalcordtoolbox/spinalcordtoolbox.git || goto :error
+
+:: Create and activate virtual environment to install SCT into
+cd spinalcordtoolbox
+python -m venv venv_sct || goto :error
+CALL venv_sct\Scripts\activate.bat || goto :error
+
+:: Upgrade pip because default p3.7 pip won't resolve dependency conflicts correctly
+python -m pip install --upgrade pip || goto :error
+
+:: Install SCT and its requirements
+pip install -r requirements.txt || goto :error
+pip install -e . || goto :error

```

Comment:
At one point I tested installing SCT in non-editable mode (`pip install .`) but that caused a mess in terms of the files outside of the `spinalcordtoolbox/` folder (`bin/`, `data/`, `contrib/`, etc.)

I foresee this being another blocker for #1526, so we should keep this in mind. (I think Nick already somewhat took this into account during his efforts for #2669?)